cmd/incus/main.go

```diff
@@ -11,7 +11,7 @@ import (
 
 	"github.com/spf13/cobra"
 
-	"github.com/lxc/incus/v6/client"
+	incus "github.com/lxc/incus/v6/client"
```

Comment:
Tiny detail but this change should not be included, `vscode` tends to do that through running `goimports`. `incus` is already the package name for that URL so putting in directly as the import name is needless duplication.

Reply:
Got it!